solutions/src/test/java/ru/sberbank/school/task13/cacheproxy/model/CacheTest.java

```diff
@@ -0,0 +1,21 @@
+package ru.sberbank.school.task13.cacheproxy.model;
+
+import ru.sberbank.school.task13.cacheproxy.annotation.Cache;
+import ru.sberbank.school.task13.cacheproxy.annotation.CacheType;
+
+import java.util.List;
+
+public interface CacheTest {
+    @Cache(getCacheType = CacheType.MEMORY)
+    String getResultUseMemoryCache();
+    @Cache(getCacheType = CacheType.FILE, key = "cache_type")
+    String getResultUseFileCache();
+    @Cache(getCacheType = CacheType.FILE, zip = true)
+    void zipFileTest();
+    @Cache(getCacheType = CacheType.MEMORY, listSize = 100)
+    List createList(int size);
+    @Cache(getCacheType = CacheType.MEMORY, identityBy = {String.class})
```

Comment:
&#39;METHOD_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1059472779>*